app/views/companies/new.html.erb

```diff
@@ -1,4 +1,9 @@
+<% if @errors %>
```

Comment:
this errors snippet is a great candidate for a partial since it's in multiple view templates